src/Symfony/Component/HttpKernel/composer.json

```diff
@@ -18,7 +18,7 @@
     "require": {
         "php": ">=5.3.9",
         "symfony/event-dispatcher": "~2.6,>=2.6.7",
-        "symfony/http-foundation": "~2.7,>=2.7.15",
+        "symfony/http-foundation": "~2.7,>=2.7.15|~2.8,>=2.8.8",
```

Comment:
this will need to be `"~2.8,>=2.8.8|~3.0,>=3.0.8|~3.1,>=3.1.2",` when merging into 3.0